### DISCUSSION TITLE
ArgumentCallback is never executed

### DISCUSSION THREAD

**Discussion by Ynverxe** [Minestom/Minestom#1958] [Help]
I was programming a custom command and after a while I see that the ArgumentCallback is never executed when the argument could not be parsed but rather the default executor of the command is executed. This also happens with the test command published on the wiki. (I tried it)

https://pastebin.com/AqUxr9ms

**Comment by mworzala** [comment]
Commands were rewritten at some point and argument callback is currently missing its implementation. I created https://github.com/Minestom/Minestom/issues/1959 from this to track it

**Comment by Mr-EmPee** [comment]
@Ynverxe I also noticed this issue, as a workaround you can catch it inside the global exception handler. Btw, I didn't like the current command system for this and other reasons so I developed my own replacement. I think that changes to the current system are bound to be slow given the high cost maintainability
